test/compile/repeat.test.ts

```diff
@@ -1,10 +1,8 @@
-import {replaceRepeaterInEncoding} from '../../src/compile/repeater';
+import {replaceRepeaterInEncoding, replaceRepeaterInFacet} from '../../src/compile/repeater';
```

Comment:
Why does this file remains here at all given there is no model left?

Reply:
We still need to replace repeater in encoding and facet in normalization. Should I move the test?

Reply:
Yes, to where the function is placed now.

Reply:
I'm moving the files and tests then